server.py

```
@@ -1045,6 +1101,161 @@ def reply_to_post(
+            _ = comment.body
+            comment_author = getattr(comment, "author", None)
+            comment_subreddit = comment.subreddit
+
+            logger.info(
+                f"Replying to comment: "
+                f"Author: {comment_author}, "
+                f"Subreddit: r/{comment_subreddit.display_name}"
+            )
+
+        except Exception as e:
+            logger.error(f"Failed to access comment {clean_comment_id}: {e}")
```

Ruff (0.14.7)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)